### DISCUSSION TITLE
Ignored domains does not appear to work.

### DISCUSSION THREAD

**Discussion by sconrad** [AdguardTeam/AdGuardHome#6035] [Q&A] [answered]
What is the correct syntax for the Ignored domains option in General Settings?

If I don't want a domain appearing in the query log from say apple.com, I tried apple.com and 'apple.com' and the queries still appear in the query log.

Any help would be appreciated.

Version: v0.107.34 running on Ubuntu VMWare VM.

**Comment by ainar-g** [comment, marked as answer]
Currently, only full domain names are supported.  So `apple.com` would **not** exclude `www.apple.com`, `developer.apple.com`, etc.  See/upvote #5720 to add support for adblock rules and/or wildcards.

**Reply by sconrad** [reply]
Thank you.

**Comment by ghost** [comment]
it would be nice to have a "no apple" list the same way there's a "no google" list
